### DISCUSSION TITLE
Is it possible to configure elsa 2.1 server to run using System.Text.Json instead of NewtonSoft

### DISCUSSION THREAD

**Discussion by HadiDghaily** [elsa-workflows/elsa-core#3990] [Q&A]
i am injecting Elsa into an already existing project which utilizes System.Text.Json but the behavior is being overridden by newtonsoft config, changing to newtonsoft is not an option . any recommendations ?

**Comment by HadiDghaily** [comment]
bump

**Comment by sfmskywalker** [comment]
Unfortunately not without refactoring everything. Elsa 3 on the other hand does use System.Text.Json, so perhaps that is an option?
